capacity4more/behat/features/group_status.feature

```diff
@@ -85,7 +85,206 @@ Feature: Group Status
     And   the "#edit-c4m-og-status-und" element should not contain "requested"
 
   @api
-  Scenario: Check deleted group as group owner
+  Scenario: Check group access by group owner
     Given I am logged in as user "alfrednobel"
-    When  I start editing group "Deleted group"
-    Then  I should not see an "#edit-c4m-og-status-und" element
+    When  I visit the dashboard of group "Requested group"
+    Then  I should have access to the page
+    And   I visit "Requested group" node of type "group"
+    And   I should not see "Access denied"
+    And   I should be allowed to edit a group "Requested group"
```

Comment:
I think this should be 'I should not see "xxx" and I should be allowed to edit group "zzz"'